src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -817,5 +817,50 @@ bool is_supported_pad(const layout& layout) {
     return (no_spatial_padding && no_batch_padding);
 }
 
+int onednn_post_ops_fusing_helpers::get_prelu_mask_from_layouts(const std::function<layout()>& get_output_layout,
+                                                                const std::function<layout(int32_t)>& get_input_layout,
+                                                                int32_t slope_input_idx) {
+    auto data_layout = get_output_layout();
```

Comment:
nit: `data_layout` is ambiguous name. What about `data_layout` -> `output_layout`?